buildspec.yml

```diff
@@ -0,0 +1,33 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - git clone https://github.com/crungehottman/exodus-lambda.git
+      - cd exodus-lambda
+      - git checkout codebuild
```

Comment:
Before merge/final review, I will remove this line.

Reply:
Is it meant to be removed, or changed to "deploy" branch?  That's what https://github.com/release-engineering/exodus-lambda/pull/48/files refers to.